inference-engine/src/gna_plugin/transformations/convert_matmul_to_pointwise_convolution.cpp

```diff
@@ -83,10 +97,29 @@ static bool Convert(std::shared_ptr<ngraph::Node> matmul_node,
     ngraph::copy_runtime_info(transpose_before, conv_node);
 
     std::shared_ptr<ngraph::Node> root_node = matmul_node;
-    if (bias != nullptr) {
-         conv_node = std::make_shared<ngraph::opset7::Add>(conv_node, bias);
-         ngraph::copy_runtime_info(transpose_before, conv_node);
-         root_node = add;
+    if (bias) {
+        auto bias_output_shape = bias->get_output_shape(0);
+        if (bias_output_shape.size() > 4) {
+            gnalog() << "bias output shape is more than 4\n";
```

Comment:
please, add bias name into the log, otherwise it'll be difficult to troubleshoot large networks.

Reply:
done